web/views.py

```
@@ -6906,3 +6909,266 @@ def delete_post(request, post_id):
+        public = request.POST.get("public") == "on"
+        if title:
+            if len(title) > 100:  # Assuming LearningMap.title has max_length=100
+                messages.error(request, "Title must be 100 characters or less")
+                return redirect("learning_map_list")
+
+            from django.db import transaction
+
+            try:
+                with transaction.atomic():
+                    new_map = LearningMap.objects.create(
+                        user=request.user, title=title, description=description, public=public
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)